qa/machine_types/schedule_subset.sh

```diff
@@ -36,6 +36,9 @@ elif [ $2 = "jewel" ] ; then
 elif [ $2 = "kraken" ] ; then
         # run kraken branch with /40 jobs
         teuthology-suite -v -c $2 -m $3 -k distro -s $4 --subset $(echo "(($(date +%U) % 4) * 7) + $1" | bc)/40 -e $5 $6
+elif [ $2 = "luminous" ] ; then
+        # run luminous branch with /40 jobs
+        teuthology-suite -v -c $2 -m $3 -k distro -s $4 --subset $(echo "(($(date +%U) % 4) * 7) + $1" | bc)/40 -e $5 $6
 else
```

Comment:
If luminous, kraken and jewel are executing the same statement inside, why can't it be combined in a single `if` ? Or is it intended ?